geomesa-spark-jts/src/main/scala/org/locationtech/geomesa/spark/WKUtils.scala

```diff
@@ -0,0 +1,54 @@
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.spark
+
+import com.vividsolutions.jts.geom.Geometry
+import com.vividsolutions.jts.io.{WKBReader, WKBWriter, WKTReader, WKTWriter}
+import org.apache.commons.pool.impl.GenericObjectPool
+
+trait ObjectPoolUtils[A] {
```

Comment:
can we add a comment to this file that it's a duplicate of the one in geomesa-utils, to prevent extra deps?